BD-2561: Personalized Paths beta [MERGE 11/6] (#6446)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md

---
nav_title: Personalized Paths 
article_title: Personalized Paths in Experiment Paths 
page_type: reference
description: "Personalized Paths is similar to Personalized Variant in campaigns and lets you automate your A/B tests."
tool: Canvas
---

# Personalized Paths in Experiment Paths

> > Personalized Paths is similar to [Personalized Variant]({{site.baseurl}}/user_guide/engagement_tools/testing/multivariant_testing/optimizations/#personalized-variant) in campaigns and lets you automate A/B tests in your Canvas.

Reviewer: "in campaigns and lets you personalize entire journeys for individual users based on conversion likelihood."

> > Personalized Paths is similar to [Personalized Variant]({{site.baseurl}}/user_guide/engagement_tools/testing/multivariant_testing/optimizations/#personalized-variant) in campaigns and lets you automate A/B tests in your Canvas.

Author: ```suggestion
> Personalized Paths is similar to [Personalized Variant]({{site.baseurl}}/user_guide/engagement_tools/testing/multivariant_testing/optimizations/#personalized-variant) in campaigns and lets you personalize entire Canvas journeys for individual users based on conversion likelihood.
```

## Prerequisites

You can use Personalized Paths in your Experiment Step when your Canvas is set to send once. Personalized Paths is not available for recurring or triggered Canvases.

{% alert important %}
Personalized Paths is currently in beta. If you are interested in participating in the beta, reach out to your customer success manager.
{% endalert %}

## Using Personalized Paths

### Step 1: Add an Experiment Path

Add an [Experiment Path]({{site.baseurl}}/user_guide/engagement_tools/canvas/canvas_components/experiment_step/) to your Canvas, then turn on **Personalized Paths**.

![][1]{: style="max-width:75%;" }

### Step 2: Configure Personalized Paths settings

Specify the conversion event that should determine the winner. If there are no conversion events available, return to the first step of Canvas setup and [assign conversion events]({{site.baseurl}}/user_guide/engagement_tools/canvas/create_a_canvas/create_a_canvas/#choose-conversion-events). Note that if you determine the winner with opens and clicks, only the first message in the path that generates opens or clicks will contribute to determining the winner.  

> Then set the **Experiment Window**. The **Experiment Window** determines how long users will be sent down all paths before choosing a single winning path for all subsequent users. The window begins when the first user enters the step.

Reviewer: before choosing "the best paths for each user in the delay group."

> Then set the **Experiment Window**. The **Experiment Window** determines how long users will be sent down all paths before choosing a single winning path for all subsequent users. The window begins when the first user enters the step.

Author: ```suggestion
Then set the **Experiment Window**. The **Experiment Window** determines how long users will be sent down all paths before choosing the best path for each user in the delay group. The window begins when the first user enters the step.
```

![][2]{: style="max-width:75%;" }

### Step 3: Determine fallback

By default, if the results of the test aren't enough to determine a statistically significant winner, all future users will be sent down the best-performing path.

Alternatively, you can select **Continue sending all future users the mix of paths**.

![][3]

This option will send future users down the mix of paths according to the percentages specified in the experiment path distribution.

![][4]

### Step 4: Launch the Canvas

Finish setting up your Canvas as needed, then launch it. When the first user has entered the experiment, you can check the Canvas to see analytics as they come in.

![][5]{: style="max-width:75%;" }

> When the experiment window passes and the experiment is complete, Braze will send users in the delay group to their respective paths.

Reviewer: optional: "to their respective paths with the highest personalized likelihood of conversion."

> When the experiment window passes and the experiment is complete, Braze will send users in the delay group to their respective paths.

Author: ```suggestion
When the experiment window passes and the experiment is complete, Braze will send users in the delay group to their respective paths with the highest personalized likelihood of conversion.
```

---
nav_title: Personalized Paths 
article_title: Personalized Paths in Experiment Paths 
page_type: reference

> description: "Personalized Paths is similar to Personalized Variant in campaigns and lets you automate your A/B tests."

Author: ```suggestion
description: "Personalized Paths is similar to Personalized Variant in campaigns and lets you personalize entire Canvas journeys based on each person's conversion likelihood."
```
